CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## 1.2.0
+
+## Features and improvements
+* Add *-sort-key layout property for circle, fill, line ([#8467](https://github.com/mapbox/mapbox-gl-js/pull/8467))
```

Comment:
Could we add a few words about what this style-spec property enables? e.g.

> Add *-sort-key layout property for circle, fill, line layers to enable sorting features within a single layer

Reply:
I would also prefer that we use the `` around style-spec property names; I'm not sure if this is a convention we've been following in practice. so that this entry would say `*-sort-key`